app/mobile/src/storybook/stories/CenterView/index.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
```

Comment:
Should probably write this in TypeScript too for consistency, otherwise might be confusing for new devs